### PR TITLE
secp256k1_ecdsa_signature_parse_der support

### PR DESCRIPTION
**Pull request by mixitka**
WIP

### CODE REVIEW

**Comment by zone117x** [comment]
Looks good so far! Just need those TODO docs and size checks, as well as unit tests.

**Comment by mixitka** [comment]
@zone117x Feedback is greatly appreciated.

**Comment by mixitka** [comment]
@zone117x Does it look good to you?

File: Secp256k1.Net/Interop.cs [hunk 1]
```diff
@@ -118,6 +118,21 @@ namespace Secp256k1Net
         void* sigin     // const secp256k1_ecdsa_signature* sigin
     );
 
+    /// <summary>
+    /// Parse a DER ECDSA signature
```
Comment (zone117x) [inline Secp256k1.Net/Interop.cs]: Can you add the full text from the original header file here? https://github.com/bitcoin-core/secp256k1/blob/e96901a4b951dfb6336114d223151dd0f5ffb89b/include/secp256k1.h#L359

Thank you!

File: Secp256k1.Net/Secp256k1.cs [hunk 2]
```diff
@@ -327,6 +329,30 @@ public bool SignatureNormalize(Span<byte> normalizedSignatureOutput, Span<byte>
             }
         }
 
+        /// <summary>
+        /// Parse a DER ECDSA signature
```
Comment (zone117x) [inline Secp256k1.Net/Secp256k1.cs]: Same comment about full description https://github.com/bitcoin-core/secp256k1/blob/e96901a4b951dfb6336114d223151dd0f5ffb89b/include/secp256k1.h#L359

**Comment by zone117x** [comment]
Looks great! Requested a couple minor doc issues.
@Xenomega want to do a sanity check on this PR as well?

File: Secp256k1.Net/Interop.cs (hunk 1 above)
Comment (mixitka) [reply Secp256k1.Net/Interop.cs]: I have added the comment as requested

File: Secp256k1.Net/Secp256k1.cs (hunk 2 above)
Comment (mixitka) [reply Secp256k1.Net/Secp256k1.cs]: I have added the comment as requested

**Comment by mixitka** [comment]
@zone117x Why was the AppVeyor build cancelled?

**Comment by zone117x** [comment]
No idea. I just triggered them to re-run.

**Comment by zone117x** [comment]
Updated package with these changes published to nuget https://www.nuget.org/packages/Secp256k1.Net/0.1.54

**Pull request merged**